docs/contributing.rst

```diff
@@ -71,15 +71,15 @@ Running the compiler tests
 ==========================
 
 The ``./scripts/tests.sh`` script executes most Solidity tests and
-runs ``aleth`` automatically if it is in the path, but does not download it,
+runs ``aleth`` automatically if it is in the path. The script does not download it,
```

Comment:
I am not sure that's true, I mean there is literally a `download_aleth` function in there

Reply:
Ok - I am not going on knowledge here. I was just tyring to clarify the intent. 

So please suggest, or better, _change_  what is more accurate. Thanks.

Reply:
Yeah, it reads as if it was wrong in the first place

Reply:
Thank you for finding this

Reply:
`download_aleth` does not always download aleth.

Reply:
Okay, I just checked and tried. The behavior is that it does NOT download aleth except when it's run in the CI. So it is basically correct :) We could add that it does not download it except in the CI, but probably not important for contributors

Reply:
So change 
> The script does not download 

to 

> The script does not always download 

?

Reply:
I would say the way you have it now is fine. The CI detail is not important for contributors